.travis.yml

```diff
@@ -1,13 +1,10 @@
 language: ruby
 
 rvm:
-  - 2.0
   - 2.1
-  - 2.2
+  - 2.2.3
+  - 2.3.0
 gemfile:
-  - gemfiles/Gemfile.ar-3.2
-  - gemfiles/Gemfile.ar-4.0
-  - gemfiles/Gemfile.ar-4.1
   - gemfiles/Gemfile.ar-4.2
```

Comment:
If the differences aren't too big I can see some convenience at having a version working on both 4.2 and 5.0.

I see only 3 small workaround. Do they bother you much?